datafusion/core/tests/sql/path_partition.rs

```diff
@@ -491,7 +491,22 @@ async fn parquet_statistics() -> Result<()> {
     // stats for the first col are read from the parquet file
     assert_eq!(stat_cols[0].null_count, Precision::Exact(1));
     // TODO assert partition column stats once implemented (#1186)
-    assert_eq!(stat_cols[1], ColumnStatistics::new_unknown(),);
+    assert_eq!(stat_cols[1], ColumnStatistics::new_unknown());
+
+    //// WITH Filter PushDown to [`ParquetExec`] ////
```

Comment:
💯  for the test, but it seems somewhat out of place in datafusion/core/tests/sql/path_partition.rs (which is testing partitioned files but this code is relevant to all)

Perhaps we could make a new test that just focused on the statistics for `ParquetExec` with pushdown filters